app/utilities/schema.py

```diff
@@ -45,18 +46,18 @@ def load_schema_file(schema_file, language_code=None):
         raise e
 
 
-def _get_schema_file_path(schema_file, language_code):
-    if language_code != 'en':
-        schema_dir = os.path.join(settings.EQ_SCHEMA_DIRECTORY, language_code)
-    else:
-        schema_dir = settings.EQ_SCHEMA_DIRECTORY
+def get_schema_path(language_code='en'):
+    return os.path.join(settings.EQ_SCHEMA_DIRECTORY, language_code)
+
 
+def get_schema_file_path(schema_file, language_code):
+    schema_dir = get_schema_path(language_code)
     return os.path.join(schema_dir, schema_file)
 
 
 def available_json_schemas():
     files = []
-    for file in os.listdir(settings.EQ_SCHEMA_DIRECTORY):
-        if os.path.isfile(os.path.join(settings.EQ_SCHEMA_DIRECTORY, file)) and file.endswith(".json"):
+    for file in os.listdir(get_schema_path(DEFAULT_LANGUAGE_CODE)):
+        if os.path.isfile(get_schema_file_path(file, DEFAULT_LANGUAGE_CODE)) and file.endswith(".json"):
```

Comment:
`get_schema_path` and `get_schema_file_path` both default to `DEFAULT_LANGUAGE_CODE`, so there is no need to pass it here.